photutils/psf/tests/test_psf_photometry.py

```diff
@@ -164,6 +165,36 @@ def test_psf_photometry_gaussian():
 
 
 @pytest.mark.skipif('not HAS_SCIPY')
+def test_psf_photometry_uncertainties():
+    """
+    Make sure proper columns are added to store uncertainties on fitted
+    parameters.
+    """
+    psf = IntegratedGaussianPRF(sigma=GAUSSIAN_WIDTH)
+    f = psf_photometry(image, INTAB, psf, param_uncert=True)
+    assert_equal(f['flux_fit_uncertainty'].all() > 0 and
+                 f['x_0_fit_uncertainty'].all() > 0 and
+                 f['y_0_fit_uncertainty'].all() > 0, True)
```

Comment:
This works. However, if might be good to be a little more specific. How large do you expect the uncertainties to be?
Testing e.g. `>0.1 & < 10` is probably good enough that the test will never fail by chance, and it would also catch if, for example, the `x_0_fit_uncertainty` column accidentally was filled with the `x` values instead of the x uncertainties.